.github/project-community-pr-assigner.yml

```diff
@@ -125,3 +117,135 @@
 
 '**/package.json':
   - team: flux
+
+"php/blueprint":
+  - team: ghidorah
+
+"php/remote-specs-validation":
+  - team: ghidorah
+
+"plugins/woocommerce-beta-tester":
+  - team: ghidorah
+
+"plugins/Woocommerce":
```

Comment:
This entry can be dropped - the following paths make this entry unnecessary I think.